### PR TITLE
add installation guide

### PR DESCRIPTION
**Pull request by haircommander**
fixes https://github.com/containers/conmon/issues/58

Signed-off-by: Peter Hunt <pehunt@redhat.com>

### CODE REVIEW

**Comment by haircommander** [comment]
@TomSweeneyRedHat PTAL

File: install.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+## Clone the repository and compile:
+
+``` bash
+git clone https://github.com/containers/conmon.git
+make
+```
+
+
+## Install to the directory you want:
+There are three options for installation, depending on your need. Each can have the PREFIX overridden. The PREFIX defaults to `/usr/local`
```
Comment (TomSweeneyRedHat) [inline install.md]: s/your need/your environment/

Comment (TomSweeneyRedHat) [reply install.md]: Would it be safe to add to the end here "`/usr/local` for most Linux distributions."  Regardless, please add an ending period.

File: install.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+- `make install` installs to `$PREFIX/bin`, usually for adding conmon to the path
```
Comment (TomSweeneyRedHat) [inline install.md]: suggest dropping "usually"

File: install.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+- `make podman` installs to `$PREFIX/libexec/podman`, which can be used to override conmon version podman uses
+- `make crio` installs to `$PREFIX/libexec/crio`, and functions similar to `make podman`
+
```
Comment (TomSweeneyRedHat) [inline install.md]: This is my windmill fight, but I'd prefer periods at the end.

File: install.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+git clone https://github.com/containers/conmon.git
+make
+```
+
+
+## Install to the directory you want:
+There are three options for installation, depending on your need. Each can have the PREFIX overridden. The PREFIX defaults to `/usr/local`
+
+- `make install` installs to `$PREFIX/bin`, usually for adding conmon to the path
+- `make podman` installs to `$PREFIX/libexec/podman`, which can be used to override conmon version podman uses
```
Comment (TomSweeneyRedHat) [inline install.md]: suggest "which is used to override the conmon version that Podman uses.

File: install.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+- `make crio` installs to `$PREFIX/libexec/crio`, and functions similar to `make podman`
```
Comment (TomSweeneyRedHat) [inline install.md]: suggest "which is used to override the conmon version that CRI-O uses."

File: install.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
+# conmon Installation Instructions
+
+In most cases, conmon should come packaged with your favorite container manager.
```
Comment (TomSweeneyRedHat) [inline install.md]: suggest s/come package/be packaged/

File: install.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+  pkg-config \
+  make \
+  runc
+```
+
+## Clone the repository and compile:
+
+``` bash
+git clone https://github.com/containers/conmon.git
+make
```
Comment (TomSweeneyRedHat) [inline install.md]: need `cd conmon`

**Comment by TomSweeneyRedHat** [comment]
Just a comment or two, great start.

**Comment by giuseppe** [comment]
LGTM

**Pull request merged**